test/associations/oneToOne/camelize.test.js

```diff
@@ -1,9 +1,10 @@
+"use strict";
+
 var it = require('it'),
     assert = require('assert'),
     helper = require("../../data/oneToOne.helper.js"),
     patio = require("index"),
     comb = require("comb"),
-    Promise = comb.Promise,
     hitch = comb.hitch;
```

Comment:
Do we use hitch in this file? I've been able to remove comb and hitch in most other tests.